docs/ignoring-files-folders-code.md

```diff
@@ -194,8 +204,15 @@ Semgrep AppSec Platform users can disable rules and rulesets through the Policie
 * [Ignore findings in Semgrep AppSec Platform](/semgrep-code/triage-remediation/#ignore-findings)
 * [Ingore findings through GitHub PR comments](/semgrep-code/triage-remediation/#ignore-findings-through-github-pr-comments)
 
+## Troubleshooting
+
+**For GitLab users**: if you use [the `SAST_EXCLUDED_PATHS` variable](https://docs.gitlab.com/ee/user/application_security/sast/#vulnerability-filters) to specify paths excluded from analysis, you may find that Semgrep doesn't honor these items. This is due to default Semgrep behavior. To explicitly include files that are typically excluded from a Semgrep scan, such as `/tests`, you must:
```

Comment:
This whole section doesn't make sense to me as it's written - are users trying to exclude or include?

I think based on what the variable is called, all of this should be written to exclude the paths.

Reply:
I might have gotten tunnel vision here: the person asking the question wanted to include their test folder, which Semgrep excludes by default. They tried to work around this by explicitly stating what to include (the test folder) using the env var, but it didn't work because Semgrep ignored the env var.

To simplify things, I will focus on excluding paths (and how to make it so that things aren't excluded) instead of how to include things.